### ISSUE TITLE
1.1.2 Tag missing

### ISSUE THREAD

**Issue by chrisrrowland** [GoogleChrome/chrome-launcher#338]
Can't say I'm an expert on how the process works but there is a 1.1.2 version listed on[ npmjs.com](https://www.npmjs.com/package/chrome-launcher/v/1.1.2) but no corresponding tag here.

**Comment by adamraine** [comment]
Yeah I think we just didn't make a github tag for that release. Looks like there are several releases that don't have tags we should update those.
